src/routes/blog/post/sql-vs-nosql/+page.markdoc

```diff
@@ -56,11 +56,11 @@ These different NoSQL types offer flexibility in data models, allowing developer
 
 ## **Flexibility and schema design in NoSQL**
 
-One major benefit of NoSQL databases is their ability to work with varied data models. Without a set structure, NoSQL databases can adjust quickly to changing requirements, which can be very useful in projects with evolving data. For example, NoSQL's flexibility makes it easy to add new data attributes on the fly, unlike SQL databases, where major changes may require extra setup.
+One major benefit of NoSQL databases is their ability to work with varied data models. Without a set structure, NoSQL databases can adjust quickly to changing requirements, which can be very useful in projects with evolving data. For example, NoSQL's flexibility makes it easy to add new data columns on the fly, unlike SQL databases, where major changes may require extra setup.
```

Comment:
Columns is incorrect here since we're talking about NoSQL

It should be attributes